packages/commonwealth/tsconfig.json

```diff
@@ -16,14 +16,9 @@
     }
   },
   "include": [
-    "./client",
```

Comment:
I'm a little worried about this being removed. If you don't want to compile these then we should use a separate tsconfig but unless I am mistaken this removes type-checking in these files which is hugely important for `yarn check-types`